tapa/verilog/xilinx/module.py

```diff
@@ -414,9 +415,10 @@ def generate_istream_ports(
                 port=self.get_port_of(port, suffix).name,
                 arg=arg_name,
             )
-
+            # peek port
+            if not enable_peek:
+                continue
             if STREAM_PORT_DIRECTION[suffix] == "input":
-                # peek port
                 if port in ignore_peek_fifos:
```

Comment:
Can you remove the ignore_peek_fifos parameter and fold it into enable_peek, so that we have a single variable controlling if peek is generated?